plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -140,43 +132,144 @@ class ServiceDestroyModal extends React.Component {
     }, REDIRECT_DELAY);
   }
 
-  render() {
-    const { isPending, onClose, open, service } = this.props;
+  getCloseButton() {
+    const { onClose } = this.props;
+
+    return (
+      <div className="row text-align-center">
+        <button
+          className="button button-primary button-medium"
+          onClick={onClose}
+        >
+          Close
+        </button>
+      </div>
+    );
+  }
+
+  getDestroyFrameworkModal() {
+    const { open, service, onClose, intl } = this.props;
+    const serviceName = service ? service.getId() : "";
```

Comment:
@orlandohohmeier yes and no I would say because not always we are expecting the service I'm guessing since the condition check was there already but I only cleaned up using ternary